model/dns/registry.yaml

```diff
@@ -16,3 +16,9 @@ groups:
           as escaped base 10 integers (\DDD). Back slashes and quotes should be escaped.
           Tabs, carriage returns, and line feeds should be converted to \t, \r, and
           \n respectively.
+      - id: dns.answers
```

Comment:
This feels like a request/response logging mechanism.

Is there a reason .NET prefers these as span attributes vs. Events?  

Note: I don't want to open the giant can of worms of Events vs. Span Attributes.  I'm just trying to understand rationale here *and* from the non-.NET portion of semantic conventions, I don't think this impacts choice of span-attribute vs. event-attribute for `dns.answers`.

Reply:
I just noticed your comment @jsuereth 

what would be the reason to report `dns.answers` in event?

Here's an example of this PR in action

<img width="1361" alt="image" src="https://github.com/user-attachments/assets/6a5b3a86-4806-4567-b556-30a13d1fcfba" />

DNS lookup takes 30ms and can fail, it's a valid span. And then there are a few addresses lookup returns, they make sense on the span that captures DNS lookup. 

Am I missing something?

Reply:
I agree the time taken to call DNS is a span, it's more whether the answers should be on the span.

I think given the direction we're now taking, we're not moving request/response to Event signal generically, so consider this comment resolved.